### PR TITLE
Makes  grenades peak

### PR DESCRIPTION
**Pull request by Lumipharon**
## About The Pull Request
Explosive grenades now blow your hand off if they explode... in your hand.

https://github.com/user-attachments/assets/d0b95138-7263-4dd0-8b2b-405216be933c

## Why It's Good For The Game
Game is borderline unplayable without this feature.
## Changelog
:cl:
add: Explosive grenades will blow your hand off if they detonate in-hand
/:cl:

### CODE REVIEW

**Review by Kuro020 — APPROVED** [review]

**Comment by despong** [comment]
If you want another funny grenade interaction mechanic you should make it so that diving onto a grenade adds you to the roundend wall of heroes like how identification dog tags does

**Comment by Tonklaatech** [comment]
Does this take into account explosive armor, IE Hlin module?

**Comment by iforgotmeword** [comment]
please make it a stat block for end of game overview
i want to see:

2 people held on to their grenades too long

File: code/game/objects/items/explosives/grenades/grenade.dm [hunk 1]
```diff
@@ -86,19 +95,20 @@
...
+///Detonation effects
 /obj/item/explosive/grenade/proc/prime()
+	if(ishuman(loc))
+		var/mob/living/carbon/human/idiot = loc
+		if(idiot.l_hand == src)
+			idiot.amputate_limb(BODY_ZONE_PRECISE_L_HAND)
+		else if(idiot.r_hand == src)
+			idiot.amputate_limb(BODY_ZONE_PRECISE_R_HAND)
+		idiot.visible_message(span_danger("[idiot]'s hand is blown into tiny pieces by the detonating [src]!"),
+		span_userdanger("You feel incredible pain and stupidity as the [src] blows your hand up."))
```
Comment (TiviPlus) [inline code/game/objects/items/explosives/grenades/grenade.dm:107]:
```suggestion
		idiot.visible_message(span_danger("[idiot]'s hand is blown into tiny pieces by [src]!"),
		span_userdanger("You feel incredible pain and stupidity as [src] blows your hand up."))
```

**Comment by Looking4bros** [comment]
no bombproofing check, but alas, you can just shove them back into whatever pouch or storage you pulled it out of to fling yourself around randomly.

**Pull request merged**